web-app/react-app/components/AuthPage/shared/Checkbox/checkbox.pcss

```diff
@@ -0,0 +1,39 @@
+.root {
+  display: flex;
+  font-size: 0.75em;
+  margin: 0;
```

Comment:
Add ` cursor: pointer; ` to get the 'Remember Me' text displayed as clickable

Reply:
@mondoreale

Reply:
[CORE-1497](https://streamr.atlassian.net/browse/CORE-1497)